mbq/client/contrib/permissions.py

```diff
@@ -458,7 +505,7 @@ def has_all_permissions(  # noqa: F811
         )
         return result
 
-    def _parse_raw_org_refs(self, raw_org_refs: Iterable[str]) -> ConvenientOrgRefs:
+    def _parse_raw_org_refs(self, raw_org_refs: List[str]) -> ConvenientOrgRefs:
```

Comment:
nit: don't we still want to be permissive on what arguments this will take in?

Reply:
Yes you are right, _but_ I am not comfortable with using `Iterable[str]` in particular, because a `str` is technically an `Iterable[str]` in mypy, so it is easy to pass in the wrong type here. Making this explicit is preferable for this instance.

Reply:
I'm with Vin here, MyPy is stupid